includes/class-wc-discounts.php

```diff
@@ -906,7 +908,7 @@ protected function get_object_subtotal() {
 		if ( is_a( $this->object, 'WC_Cart' ) ) {
 			return wc_add_number_precision( $this->object->get_displayed_subtotal() );
 		} elseif ( is_a( $this->object, 'WC_Order' ) ) {
-			return wc_add_number_precision( $this->object->get_subtotal() );
+			return wc_add_number_precision( $this->object->get_subtotal() ) + round( $this->object->get_total_tax(), wc_get_price_decimals() );
```

Comment:
This needs to check if we need tax to be added to subtotal, based on store settings.

Reply:
Thanks, updated.